sp800-63b/sec5_authenticators.md

```diff
@@ -359,7 +359,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 
 If matching is performed centrally:
 
-* Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
+* Use of the biometric SHALL be limited to a single, specific device that is identified using approved cryptography.
```

Comment:
Confirmed with @enewt that where  I was heading is correct.  Recommend dropping the second bullet and changing the 1st to:  Use of the biometric SHALL be bound tightly to a trusted device that is identified using approved cryptography.